includes/rest-api/rest-api-endpoint-registration.php

```diff
@@ -680,3 +724,60 @@ function content_model_multi_option_exists( array $names, string $current_choice
 	}
 	return false;
 }
+
+/**
+ * Saves a taxonomy.
+ *
+ * @param array $params Parameters passed from the taxonomy form.
+ * @param bool  $is_update True if `$params` came from a PUT request.
+ * @return object|WP_Error
```

Comment:
```suggestion
 * @return array|WP_Error
```

It looks like `$taxonomy` is an array

Reply:
Good catch, thanks — not sure why PhpStorm didn't infer that. 🤷